components/devtools/actors/root.rs

```diff
@@ -95,17 +95,25 @@ struct ListProcessesResponse {
     processes: Vec<ProcessForm>,
 }
 
+#[derive(Default, Serialize)]
+pub struct DescriptorTraits {
+    pub(crate) watcher: bool,
+    pub(crate) supportsReloadDescriptor: bool,
```

Comment:
I think it would be good to start using serde renaming so that this file can remove `allow(non_snake_case)` one day.

```suggestion
    #[serde(rename(serialize = "supportsReloadDescriptor"))]
    pub(crate) supports_reload_descriptor: bool,
```

Reply:
That's awesome, I didn't know serde could do that! I found out you can even rename all the fields of a struct using camelCase [automatically](https://serde.rs/attr-rename.html), which seems more robust to avoid spelling mistakes in the future.

Reply:
Oh, great find!